modules/dictionary/php/datadictrowprovisioner.class.inc

```diff
@@ -69,6 +69,14 @@ class DataDictRowProvisioner extends \LORIS\Data\ProvisionerInstance
                     $status = 'Empty';
                 }
 
+                $cohorts = $DB->pselectCol(
+                    "SELECT DISTINCT tb.CohortID FROM test_names tn
+                        JOIN test_battery tb ON tn.Test_name=tb.Test_name
+                        WHERE tn.Test_name=:tn
+                    ORDER BY tb.CohortID",
+                    ['tn' => $cat->getName()]
+                );
```

Comment:
According to this statement, several variables named `$cohorts` refer to cohort IDs, hence I would prefer them to be named `$cohort_ids`.

I also see that some of them are documented as of type `?string[]`, but cohort IDs are nullable integers in the database. This is probably inaccurate ?

Reply:
Thank you! I have updated it to use the cohort name instead of the CohortID, as candidate_list does. It makes more sense as now in the list you see the titles instead of the IDs!